src/analytics/src/main/scala/osmesa/analytics/oneoffs/StreamingAOIMonitor.scala

```diff
@@ -1,234 +1,210 @@
+
+import cats.data.Validated
 import cats.implicits._
-import com.monovore.decline.{CommandApp, Opts}
+import com.monovore.decline.{Argument, CommandApp, Opts}
 import geotrellis.vector._
 import org.apache.spark.internal.Logging
-import org.apache.spark.sql.{Dataset, SparkSession}
+import org.apache.spark.sql._
 import org.apache.spark.sql.functions.{collect_list, collect_set, count, explode, size, udf}
-import org.locationtech.geomesa.spark.jts.{udf => _, _}
+import org.locationtech.geomesa.spark.jts._
+import org.locationtech.jts.geom.{GeometryFactory, PrecisionModel}
 import org.locationtech.jts.{geom => jts}
 import org.locationtech.jts.geom.prep._
+import org.locationtech.jts.io.WKBReader
 import osmesa.analytics.Analytics
+import osmesa.analytics.oneoffs.Interval._
 import osmesa.analytics.stats._
 import osmesa.analytics.stats.functions._
 import vectorpipe.functions._
-import vectorpipe.sources.{AugmentedDiffSource, Source}
-import vectorpipe.util.DBUtils
+import vectorpipe.sources.Source
 
-import java.net.URI
-import java.sql._
+import scala.collection.mutable.ListBuffer
+import scala.util.Properties
 
 // These only required for test getAreaIndex implementation
 import geotrellis.vector.io._
 import geotrellis.vector.io.json._
 import spray.json._
 
-object StreamingAOIMonitor extends CommandApp(
-  name = "streaming-aoi-monitor",
-  header = "Streaming AOI Monitor",
-  main = {
-    val intervalOpt =
-      Opts
-        .option[String]("interval", help = "Period of time to aggregate over (d=daily, w=weekly)")
-        .validate("Must be one of 'd', 'w'") { arg: String => arg.length == 1 && "dw".contains(arg.apply(0)) }
-
-    val databaseUrlOpt =
-      Opts
-        .option[URI](
-        "database-url",
-        short = "d",
-        metavar = "database URL",
-        help = "URL of database containing AOI table (default: DATABASE_URL environment variable)"
-      )
-      .orElse(Opts.env[URI]("DATABASE_URL", help = "The URL of the database"))
-
-    val augmentedDiffSourceOpt =
-      Opts
-        .option[URI](
-        "augmented-diff-source",
-        short = "a",
-        metavar = "uri",
-        help = "Location of augmented diffs to process"
-      )
-
-    val startSequenceOpt =
-      Opts
-        .option[Int](
-        "start-sequence",
-        short = "s",
-        metavar = "sequence",
-        help = "Starting sequence. If absent, the current (remote) sequence will be used."
-      )
-      .orNone
-
-    val endSequenceOpt =
-      Opts
-        .option[Int](
-        "end-sequence",
-        short = "e",
-        metavar = "sequence",
-        help = "Ending sequence. If absent, this will be an infinite stream."
-      )
-      .orNone
-
-    (intervalOpt, databaseUrlOpt, augmentedDiffSourceOpt, startSequenceOpt, endSequenceOpt).mapN {
-      (interval, databaseUri, augmentedDiffSource, startSequence, endSequence) =>
-      val appName = "StreamingAOIMonitor"
+object StreamingAOIMonitor
+    extends CommandApp(
```

Comment:
Moving this to a separate line and reindenting made this diff very hard to read.  These types of cosmetic changes should probably be put off until after substantive comments are addressed in the future.

Reply:
Yeah, I made note of this in the commit log and PR description. Totally understood. I couldn't untangle it from in flight commits all that well so I left it in. Don't plan to let it happen in the future.

> An automated scalafmt snuck its way in there in the
middle of my efforts. Untangling it was a bit of a
headache so I just left it, even though it should
be a separate commit.